dali/python/backend_impl.cc

```diff
@@ -1075,7 +1089,7 @@ PYBIND11_MODULE(backend_impl, m) {
     .def("SetExternalTLInput",
         [](Pipeline *p, const string &name, const TensorList<CPUBackend> &tl,
            py::object /*cuda_stream*/) {
-          p->SetExternalInput(name, tl, 0);
+          p->SetExternalInput(name, tl, 0, true);
```

Comment:
I assume it is fine to pass a false here as well (or rather it just doesn't matter)?

Reply:
But it's more aligned with the docs that claim to be blocking.

Reply:
It is CPU so it should sync anyway, so it is better to pass the true to be aligned with what happens under the hood.